cachito/web/utils.py

```diff
@@ -1,7 +1,32 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
+from collections import OrderedDict
+
 from flask import request, url_for
 
 
+def deep_sort_icm(orig_item):
+    """
+    Return a new element recursively sorted in ascending order.
+
+    The function for sorting image content manifests
+
+    :param orig_item: Original dict or list
```

Comment:
(optional): I'd rename this as `:param orig_item: Original content manifest to be sorted`

Reply:
Done, thank you!